UCSLibraries/src/main/java/it/cnr/iit/xacml/wrappers/PolicyWrapper.java

```
@@ -284,8 +292,7 @@ private static PolicyType unmarshalPolicyType( String policy ) {
 
     private static String marshalPolicyType( PolicyType policy ) {
         try {
-            return JAXBUtility.marshalToString( PolicyType.class, policy, PolicyTags.POLICY,
-                JAXBUtility.SCHEMA );
+            return JAXBUtility.marshalToString( PolicyType.class, policy, PolicyTags.POLICY, JAXBUtility.SCHEMA );
         } catch( JAXBException e ) {
             log.severe( String.format( MSG_ERR_MARSHAL, e.getMessage() ) );
```

may be better to throw an exception?

---

yes